connector_jira/models/jira_backend/common.py

```diff
@@ -295,7 +297,7 @@ def _inverse_delete_analytic_line_from_date(self):
         )
 
     @api.multi
-    def _run_background_from_date(self, model, from_date_field,
+    def _run_background_from_date(self, model, from_date_field, force,
```

Comment:
not a blocker but I'd keep it as kw arg here

Reply:
sorry, didn't get you. Add `kwargs` or move `force` to kwargs?

Reply:
the second

Reply:
sounds reasonable